### DISCUSSION TITLE
Error upgrading from 2.3.190 to 2.3.200

### DISCUSSION THREAD

**Discussion by fcblank** [Security-Onion-Solutions/securityonion#9613] [Unsupported Versions]
Hi, I upgraded so from 2.3.190 to 2.3.200 on a standalone system and nearly to the end of the upgrade after successfully completing the state.highstate the following error was thrown (copy from soup.log): **Rendering SLS 'base:logstash' failed: Jinja variable 'None' has no attribute 'items'**
SecutityOnion seems to be working ok, though.
Is this a critical error I should worry about?

```    Data failed to compile:
----------
    Rendering SLS 'base:logstash' failed: Jinja variable 'None' has no attribute 'items'
/var/cache/salt/minion/files/base/logstash/map.jinja(7):
---
[...]
{% set REDIS_NODES = [] %}
{% set mainint = salt['pillar.get']('host:mainint') %}
{% set localhostip = salt['grains.get']('ip_interfaces').get(mainint)[0] %}
{% if role in ['so-node', 'so-standalone', 'so-managersearch'] %}
  {% set node_data = salt['pillar.get']('logstash:nodes') %}
  {% for node_type, node_details in node_data.items() | sort %}    <======================
    {% if node_type in ['manager', 'managersearch', 'standalone', 'receiver' ] %}
      {% for hostname in node_data[node_type].keys() %}
        {% do REDIS_NODES.append({hostname:node_details[hostname].ip}) %}
      {% endfor %}
    {% endif %}
[...]
---
...
...
Soup failed with error 1: Unhandled error
```
Technical information

```- Security Onion version 2.3.200
- on-prem deployment, Internet access
- network installation
- installed on CentOS 7 some time ago (maybe 2.3.40?)
- one node (standalone)
- HP380 gen10 2 sockets (Intel(R) Xeon(R) Silver 4110 CPU @ 2.10GHz), 32 numa nodes, 2TB RAM
- so-status shows all services are running
- SOC Grid page shows no failures
```

Many Thanks in advance for your help

**Comment by cm-ops** [comment]
Check the salt-mine - `salt-call mine.get '*' network.ip_addrs` (should output the name and IP), `salt-call pillar.get host:mainint` (will output interface name), and `salt-call grains.get ip_interfaces` (will show all interfaces with IP). If any of them are not populated try `salt-call mine.update`.

**Reply by fcblank** [reply]
thanks for your hints but the outputs of the commands is ok, t.i. all are populated

**Reply by cm-ops** [reply]
Okay, if they are all populated and you run `so-checkin` and if that completes without error, you should be fine.
